nncf/openvino/graph/nncf_graph_builder.py

```diff
@@ -184,14 +184,20 @@ def create_nncf_graph(model: ov.Model) -> NNCFGraph:
                     }
 
                     if metatype == OVMatMulMetatype:
+                        node_inputs = node.inputs()
                         attribute_names = ["transpose_a", "transpose_b"]
                         node_attributes = node.get_attributes()
-                        transpose = node_attributes[attribute_names[const_port_id]]
-                        const_attrs[const_port_id]["transpose"] = transpose
+                        const_transpose_name = attribute_names.pop(const_port_id)
```

Comment:
It looks like you can implement without `pop` here as well:

```
const_transpose_name = attribute_names[const_port_id]
act_attrs["transpose"] = node_attributes[attribute_names[act_port_id]]
```

Reply:
Thanks, updated